packages/in-memory-key-manager/test/InMemoryKeyManager.test.ts

```diff
@@ -1,33 +1,41 @@
 import { createInMemoryKeyManager } from '@src/InMemoryKey';
 import { util } from '@src/.';
-import { loadCardanoSerializationLib } from '@cardano-sdk/cardano-serialization-lib';
+import { CardanoSerializationLib, CSL, loadCardanoSerializationLib } from '@cardano-sdk/cardano-serialization-lib';
+import { Cardano } from '@cardano-sdk/core';
 import { KeyManagement } from '@cardano-sdk/wallet';
-import CardanoSerializationLib from '@emurgo/cardano-serialization-lib-nodejs';
 
 describe('InMemoryKeyManager', () => {
   let keyManager: KeyManagement.KeyManager;
-  let cardanoWasm: typeof CardanoSerializationLib;
+  let csl: CardanoSerializationLib;
 
   beforeEach(async () => {
-    cardanoWasm = await loadCardanoSerializationLib();
+    csl = await loadCardanoSerializationLib();
     const mnemonic = util.generateMnemonic();
     keyManager = createInMemoryKeyManager({
+      csl,
       mnemonic,
+      networkId: Cardano.NetworkId.testnet,
       password: '123'
     });
-    expect(keyManager.publicKey).toBeInstanceOf(CardanoSerializationLib.PublicKey);
+    expect(keyManager.publicKey).toBeInstanceOf(CSL.PublicKey);
```

Comment:
This passes because both `csl` and `CSL` are the same in nodejs environment, as CSL is exported from nodejs version. However a better test would be  `.toBeInstanceOf(csl.PublicKey)`

Reply:
Done in https://github.com/input-output-hk/cardano-js-sdk/pull/95/commits/12f0171f8cc9dcd73a2ed9426bd92e65887ea538